h/static/styles/partials-v2/_search.scss

```diff
@@ -117,18 +114,20 @@
 }
 
 .search-result-bucket__annotation-cards {
-  flex-grow: 1;
```

Comment:
I'm not sure why this was here, we don't want annotation cards to grow wider than 500px